windows/webview.h

```diff
@@ -27,6 +27,11 @@ enum class WebviewPermissionKind {
 
 enum class WebviewPermissionState { Default, Allow, Deny };
 
+struct WebviewHistoryChanged {
+  BOOL canGoBack;

```

Comment:
According to the Google C++ styleguide, it should be named `can_go_back` (see https://google.github.io/styleguide/cppguide.html#Variable_Names)